### ISSUE TITLE
Implement CKEditor language plugin

### ISSUE THREAD

**Issue by jonaslepsoy** [enonic/xp#7386]
Text readers (like VoiceOver for Mac or Narrator for Windows) don't work too good if the language on the page (the `lang` attribute) is incorrect. This is normally handled just fine by the language settings on a particular content if the developers have done their job.

A problem can arise when there is multiple languages on the same page, for example a quote in another language. To fully support text readers, the editor should be able to designate the language (i.e. the `lang` attribute) for any text when using the rich text editor without resorting to editing the raw HTML.

CKEditor has a language plugin that is perfect for this: https://ckeditor.com/cke4/addon/language

![ckeditor-language-sample_0](https://user-images.githubusercontent.com/5877868/64588275-3a6e9880-d3a2-11e9-9529-dd39120615c0.png)

Supporting changing language on a page is a requirement in WCAG 2.0, success criterion 3.1.2 Language of parts: https://www.w3.org/TR/2008/REC-WCAG20-20081211/#meaning-other-lang-id

**Comment by jonaslepsoy** [comment]
Any news on this?

**Comment by sigdestad** [comment]
Interesting solution.
We have not concluded yet as we will need to provide some way to manage available languages. Filtering this is already on the "features" required. Will update this when we have news

**Comment by alansemenov** [comment]
We have added this to our backlog. Currently not prioritised.
